assets/js/components/Dashboard/TimePicker.tsx

```diff
@@ -13,28 +21,31 @@ interface TimePickerProps {
 const TimePicker = ({ selectedTime, onChange, id }: TimePickerProps) => {
   const [showOverlay, setShowOverlay] = useState(false);
   const ref = useRef(null);
-
   const selectedMoment = moment(selectedTime, "h:mm A");
 
   return (
     <>
-      <Form.Control
-        id={id}
-        ref={ref}
-        type="time"
-        value={selectedTime}
-        onChange={(input) =>
-          onChange(moment(input.target.value, "HH:mm").format("HH:mm"))
-        }
-        onKeyUp={(e) => {
-          if (e.key === "Enter") {
-            setShowOverlay(true);
+      <InputGroup className="time-picker" ref={ref}>
+        <Form.Control
+          id={id}
+          type="time"
+          value={selectedTime}
+          onChange={(input) =>
+            onChange(moment(input.target.value, "HH:mm").format("HH:mm"))
           }
-        }}
-      />
+          onKeyUp={(e) => {
+            if (e.key === "Enter") {
+              setShowOverlay(true);
+            }
+          }}
+        />
+        <Button onClick={() => setShowOverlay(true)}>
+          <Clock />
+        </Button>
+      </InputGroup>
       <Overlay
         rootClose
-        rootCloseEvent="mousedown"
+        rootCloseEvent="click"
```

Comment:
`DatePicker` uses `rootCloseEvent="mousedown"` — I'm guessing these are not supposed to be different?

Reply:
This was an unnecessary change. Reverting.